search/factory.py

```diff
@@ -21,7 +20,6 @@ def create_ui_web_app() -> Flask:
 
     app = Flask('search')
     app.config.from_pyfile('config.py')
-    app.url_map.converters['arxiv'] = ArXivConverter
```

Comment:
It seems like this needs a replacement, allows for some nice syntax sugar for `/abs/` urls, but maybe not necessary for search.

Reply:
This is now added by base, see https://github.com/cul-it/arxiv-base/blob/master/arxiv/base/__init__.py#L48 ; so available for any app that uses base.